hazelcast/src/main/java/com/hazelcast/map/impl/record/Record.java

```diff
@@ -19,7 +19,7 @@
 import com.hazelcast.nio.serialization.Data;
 
 /**
- * @param <V>
+ * @param <V> the type of value which is in the Record
```

Comment:
Afaik we don't need this type parameter at all. I have not seen a single usage of of Record with a concrete type other then Data or object.